database/initAutoIncrement.js

```diff
@@ -0,0 +1,7 @@
+use topBunk;
+db.counters.drop();
+db.counters.insert({
+  "id" : "listings",
+  "reference_value" : null,
+  "seq" : 10000000
+});
```

Comment:
Same as comment on `postgresSeed.js`, suggest new line at end of file.

Reply:
Done.